modules/jupyter/jupyter_config/config-selfauth-autopilot.yaml

```diff
@@ -88,6 +88,7 @@ singleuser:
   extraEnv:
     # Used for GCSFuse to set the ephemeral storage as the home directory. If not set, it will show a permission error on the pod log when using GCSFuse.
     JUPYTER_ALLOW_INSECURE_WRITES: "true"
+    CLOUDSQL_INSTANCE_CONNECTION_NAME: ${cloudsql_instance_connection_name}
```

Comment:
btw. this is going to add the CLOUDSQL env var and secrets for the Jupyter-only installation (not RAG)

Reply:
(likewise for the secret mounts)

Reply:
^ we spoke offline -- I meant that jupyer only installs will set these environment varibles

Reply:
Made these variables default to empty string. If not explicitly passed, the env var will be empty.